stablehlo/dialect/TypeInference.h

```diff
@@ -163,6 +167,9 @@ LogicalResult inferSortOp(
     Region& comparator,
     SmallVectorImpl<ShapedTypeComponents>& inferredReturnShapes);
 
+LogicalResult verifySortOp(Optional<Location> location, ValueRange inputs,
+                           uint64_t dimension, Region& comparator);
+
```

Comment:
How about ordering these functions alphabetically, i.e. first all `inferFooOp` functions and then all `verifyFooOp` functions?

Reply:
Thinking about same thing, resolved in next commit.